test/functional/test_cli_commands.py

```diff
@@ -395,8 +399,8 @@ def test_infrasim(self):
         """
         CLI test: "infrasim -h" navigates next level command usage
         """
-        p = r'\{([a-zA-Z,]+)\}'
-        cmd_next_level = ["node", "chassis", "config", "init", "version"]
+        p = r'\{([a-zA-Z_,]+)\}'
+        cmd_next_level = ["node", "chassis", "config", "init", "version", "global_status"]
```

Comment:
[minor]embedding a dash looks less consistent than other argument, like node, chassis, config, etc. can we make it a "infrasim global status"